source/funkin/editors/character/CharacterEditor.hx

```diff
@@ -325,7 +325,20 @@ class CharacterEditor extends UIState {
 		else FlxG.switchState(new CharacterSelection());
 	}
 
-	function _file_new(_) {
+	public function _file_new(_) {
+		openSubState(new SaveSubstate('
+			<!DOCTYPE codename-engine-character>
+			<character isPlayer="false" flipX="false" holdTime="6.1" color="#AF66CE">
```

Comment:
Make this not have holdTime="6.1" since it would cause a lot of characters to have that, when they may not want it

Reply:
Idk how to add people's suggestions to pull requests but yeah it's a good idea

Reply:
Just push a commit to your branch and it should update

Reply:
Alr

Reply:
> Make this not have holdTime="6.1" since it would cause a lot of characters to have that, when they may not want it

6.1's just the default cos it's on Daddy Dearest

You can change it later

Reply:
Yea but imo i would remove it from the template, since some people dont know what it means

Reply:
Not even I know what it does cos I've barely seen it do anything really

Reply:
> Yea but imo i would remove it from the template, since some people dont know what it means

So what should the default be because I cannot leave it blank

Reply:
Alrighty (I'll just set the thing to 4 muahahaha)